facia/public/facebook_ia_remote.html

```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex">
+    <script>
+    </script>
+</head>
+<body style="margin:0">
+<div id="c" style="position:absolute;top:0;left:0;bottom:0;right:0;">
+
+    <script>
+        (function () {
+
+            function parseParameters() {
+                var script = document.createElement("script");
+                script.id = "ad-slot-id";
+
+                var args = window.location.hash.substr(1).split('&');
+
+                for (var i = 0; i < args.length; ++i) {
+                    var arg = args[i].split("=");
+                    var argName = arg[0];
+                    var argValue = decodeURIComponent(arg[1]);
+
+                    var currValue = script.getAttribute(argName);
+                    if (currValue) {
+                        script.setAttribute(argName, currValue + "," + argValue);
+                    } else {
+                        script.setAttribute(argName, argValue);
+                    }
+                }
+                document.getElementsByTagName("div")[0].appendChild(script);
+
+                return document.querySelectorAll("script[id$='" + script.id + "']");
+            }
+
+            var networkId = "59666047";
+
+            var scripts = parseParameters();
+
+            var slot = scripts[scripts.length - 1];
+
+            var slotName = "ad-slot-" + scripts.length;
+
+            var guardian = {} ;
+
+            function toTitleCase(str) {
+                return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+            }
+
+            function exposeMetaDataForKrux() {
+                guardian = { 'config' : {
+                    'page' : {
+                        'pageId' : slot.getAttribute("url"),
+                        'edition' : toTitleCase(slot.getAttribute("edition")),
+                        'seriesId' : slot.getAttribute("se"),
+                        'contentType' : toTitleCase(slot.getAttribute("ct")),
+                        'keywordIds' : slot.getAttribute("keywordIds"),
+                        'keywords' : slot.getAttribute("k").split(',').map(function (x) { return toTitleCase(x)}),
+                        'authorIds' : slot.getAttribute("authorIds"),
+                        'blogIds' : slot.getAttribute("bl"),
+                        'referrer' : 'facebook-ia',
+                        'section' : slot.getAttribute("section")
+                    }
+                }
+                };
+            }
+
+            function loadKrux() {
+                var krux = document.createElement('script');
+                krux.setAttribute("class", "kxct");
+                krux.setAttribute("data-id", "KS_cfybw");
```

Comment:
Once we meet with Steven, he can show you how to create a new control tag for this. This line of code and one other (where it says "KS_cfyb") can be replaced with the new id. Lets do this before we merge this in.